metadata/src/test/java/org/apache/kafka/controller/QuorumControllerTest.java

```diff
@@ -717,7 +717,7 @@ public void testUnregisterBroker() throws Throwable {
                     setBrokerId(0).
                     setClusterId(active.clusterId()).
                     setIncarnationId(Uuid.fromString("kxAT73dKQsitIedpiPtwBA")).
-                    setFeatures(brokerFeatures(MetadataVersion.IBP_3_0_IV1, MetadataVersion.IBP_3_8_IV0)).
+                    setFeatures(brokerFeatures(MetadataVersion.IBP_3_0_IV1, MetadataVersion.IBP_3_9_IV0)).
```

Comment:
Note that ideally, we should use MetadataVersion.latestTesting() here. However, explicitly using IBP_3_9_IV0 works too since it's the latest MV.